foundational-dotnet-support-matrix.md

```diff
@@ -0,0 +1,15 @@
+
+# Foundational .NET Support
+
+| Dimension       | Supported Version | Last Changed | Next Change [^next-change] |
+|-----------------|-------------------|--------------|----------------------------|
+| .NET 7          | >= 7.0.12         | 2022-11-08   | 2024-05-14                 |
+| .NET 6          | >= 6.0.23         | 2021-11-08   | 2024-11-12                 |
```

Comment:
We just target "net6.0" which means it'll work on 6.0.0+. I think it would be reasonable for us to ask customers to upgrade to the latest patch of a runtime though - that's something we don't have in the policy at the moment.

We should also add a line for .NET Framework though - we support 4.6.2 at the moment, and will do until at least 2027-01-12 (see https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework).

We also target .NET Standard - either 2.0 or 2.1. That's a fiddly one to describe, and I don't know when we'd stop supporting it - lots of open source projects target that, so it's useful if they need to depend on us. It's not a runtime as such, but an interface for runtimes... I'd be happy not including it on this page for now.

Reply:
@coryan I believe we can close this PR since Jon wrote and submitted the definitive new version.
https://github.com/google/oss-policies-info/pull/25